### PR TITLE
RAC-4817 - move and enable converted APIv2 scripts into api-cit.

### PR DESCRIPTION
**Pull request by hohene**
This move will allow them to run in smoke tests as the directory gets traversed.
Update user_tests.py to include smoke=True
Remove the outdated file nodes_tests_fit.py.

### CODE REVIEW

**Comment by hohene** [comment]
test this please

**Comment by hohene** [comment]
test this please

**Comment by hohene** [comment]
test this please

**Comment by larry-dean** [comment]
test this please

**Comment by hohene** [comment]
@johren @larry-dean @keedya @stuart-stanley @derrickostertag

File: test/tests/api-cit/v2_0/nodes_tests.py [hunk 1]
```diff
@@ -464,39 +452,63 @@ def test_node_get_obm_by_node_id(self):
...
+            self.assertEqual(404, e.status, msg='unexpected response {0}, expected 404'.format(e.status))
 
     @depends(after='test_node_get_obm_invalid_node_id')
     def test_clean_up_test_nodes(self):
         # Clean up the added test nodes
         self.assertEqual(0, self.delete_temp_nodes(), msg="Failed to clean up test nodes from script")
+
+    @classmethod
+    def clear_nodes(self):
+        # """ Clear the temp created nodes if any exist """
```
Comment (derrickostertag) [inline test/tests/api-cit/v2_0/nodes_tests.py]: does clear_nodes() actually delete the node entry?

Comment (hohene) [inline test/tests/api-cit/v2_0/nodes_tests.py]: Yes, it does delete it from the node_list via  Api().nodes_del_by_id(identifier=uuid).

File: test/tests/api-cit/v2_0/lookups_tests.py [hunk 2]
```diff
@@ -28,14 +28,14 @@ def setUpClass(cls):
...
-        cls.id = ""
+        cls.lookup_id = ""
         cls.patchedNode = {"node": "666"}
 
     def test_check_lookups(self):
         # """ Testing GET:/lookups """
         Api().lookups_get()
         rsp = self.__client.last_response
-        logs.debug("Lookup list: %s", dumps(rsp.data, indent=4))
+        # logs.debug("Lookup list: %s", dumps(rsp.data, indent=4))
```
Comment (stuart-stanley) [inline test/tests/api-cit/v2_0/lookups_tests.py]: nit: should this still be commented out?

File: test/tests/api-cit/v2_0/lookups_tests.py [hunk 3]
```diff
@@ -99,12 +99,12 @@ def test_post_lookup(self):
         Api().lookups_post(self.lookup)
         rsp = self.__client.last_response
         self.assertEqual(201, rsp.status, msg=rsp.reason)
-        self.id = str(loads(rsp.data).get('id'))
-        logs.debug("ID is %s", self.id)
+        self.lookup_id = str(loads(rsp.data).get('id'))
+        logs.debug("ID is %s", self.lookup_id)
```
Comment (stuart-stanley) [inline test/tests/api-cit/v2_0/lookups_tests.py]: nit: log ("lookup ID is...")

File: test/tests/api-cit/v2_0/obm_tests.py [hunk 4]
```diff
@@ -57,5 +57,5 @@ def test_setup_snmp_obm_api2(self):
     @depends(after='test_setup_snmp_obm_api2')
     def test_check_snmp_obm_settings(self):
         # """ Checking SNMP OBM settings GET:/nodes """
-        self.assertNotEqual(len(obmSettings().check_nodes(service_type='snmp-obm-service')), 0,
-                            msg='There are missing SNMP OBM settings!')
+        self.assertEqual(len(obmSettings().check_nodes(service_type='snmp-obm-service')), 0,
```
Comment (stuart-stanley) [inline test/tests/api-cit/v2_0/obm_tests.py]: Either the assert is wrong or the message is wrong? We assert that the number of nodes with snmp-obm-service must be 0 but then state that there SNMP OBM settings are missing. I.E., we will blow up when there ARE settings!

Comment (hohene) [inline test/tests/api-cit/v2_0/obm_tests.py]: The assert and message are proper, even though it looks backwards.  This 'check_nodes' will return a list of nodes that have errors (ie. missing SNMP settings).  if the list is empty (length = 0), that means no errors. 
If the list is not empty, the error message is proper that the listed nodes do not have expected SNMP settings.

Comment (stuart-stanley) [inline test/tests/api-cit/v2_0/obm_tests.py]: ok

File: test/tests/api-cit/v2_0/lookups_tests.py (hunk 2 above)
Comment (hohene) [inline test/tests/api-cit/v2_0/lookups_tests.py]: added back in at debug_7 level, output can be extensive, so only when you really need it.

File: test/tests/api-cit/v2_0/lookups_tests.py (hunk 3 above)
Comment (hohene) [inline test/tests/api-cit/v2_0/lookups_tests.py]: fixed

**Pull request merged**